SeleniumPracticalExercise/PageObjects/VerifyEmployee.cs

```diff
@@ -0,0 +1,69 @@
+﻿using NUnit.Framework;
+using OpenQA.Selenium;
+using OpenQA.Selenium.DevTools.V125.CSS;
+using OpenQA.Selenium.DevTools.V125.Page;
+using SeleniumExtras.WaitHelpers;
+using SeleniumPracticalExercise.PageObjects.Common;
+using static SeleniumPracticalExercise.PageObjects.Common.BasePageLocal;
+
+
+namespace SeleniumPracticalExercise.PageObjects
+{
+    public class VerifyEmployee : BasePageLocal
+    {
+        private readonly By EmployeeId = By.CssSelector("div.oxd-grid-2 input");     // EMployee ID # Locator
+        private readonly By SaveButton = By.CssSelector("button[type='submit']");    // Save Button Locator
+        private readonly By PIMButton = By.CssSelector("a[href='/web/index.php/pim/viewPimModule']");   // PIM Locator
+        private readonly By TxtBoxEmployeeID = By.CssSelector("div.oxd-input-group input.oxd-input");  // EmployeeID TextBox locator 
+        private readonly By SubmitButton = By.CssSelector("button[type='submit']");  // Submit Locator
+        private readonly By DisplayedID = By.XPath("//div[contains(@class,'oxd-table-row--clickable')]//div[text()=''][2]/div");  // search result displayed ID
+        private readonly By FirstNameDisplay = By.XPath("//div[contains(@class,'oxd-table-row--clickable')]//div[text()=''][3]/div");  //search result displayed firstname
+        private readonly By LastNameDisplay = By.XPath("//div[contains(@class,'oxd-table-row--clickable')]//div[text()=''][4]/div");   //search result displayed lastname
```

Comment:
You can do these more simply with a CSS selector

Reply:
not sure how?
I thought you were testing us since it was a table and it took me 2 hours just to come up with those xpaths.... and I was proud of myself 😭

Reply:
We went through this last Thu

Reply:
I know you showed me how to do it in CSS but I forgot.
I tried 
$$("div.oxd-table-cell div")
and tried to add the [1] but didn't get anything.

I'll keep trying and come back to it later

Reply:
Don't forget to come back to this